addons/zeus/XEH_postInit.sqf

```diff
@@ -14,3 +14,26 @@ QGVAR(GlobalSkillAI) addPublicVariableEventHandler FUNC(moduleGlobalSetSkill);
 [QGVAR(modulePatrolArea), CBA_fnc_taskPatrol] call CBA_fnc_addEventHandler;
 [QGVAR(moduleSearchNearby), CBA_fnc_searchNearby] call CBA_fnc_addEventHandler;
 [QGVAR(moduleSearchArea), CBA_fnc_taskSearchArea] call CBA_fnc_addEventHandler;
+
+// Editable object commands must be ran on server, this events are used in the respective module
+if (isServer) then {
+    [QGVAR(addObjects), {
+        params ["_objects", "_curator"];
+
+        if !(isNull _curator) exitWith { _curator addCuratorEditableObjects [_objects, true]; };
+
+        {
+            _x addCuratorEditableObjects [_objects, true];
+        } forEach allCurators;
+    }] call CBA_fnc_addEventHandler;
+
+    [QGVAR(removeObjects), {
+        params ["_objects", "_curator"];
```

Comment:
@SilentSpike 
Do you mean:
`params ["_objects", ["_curator", objNull]];`
so that the `exitWith` thing works as alternative syntax?
Alternativeley change `!isNull _curator` to `!isNil "_curator"`

same above